bin/autoUpdateRepo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#:Title: autoUpdateRepo.sh
+#:Author: Alan Coleman
+
+#changes directory to folder containing repositories
+cd ~/gitfolder
```

Comment:
should this be pointing to an ENV variable e.g. cd $ELK_POC_HOME?

Reply:
at the moment, environmental variables in the script, do not work when ran from crontab. Balaji had added fixing this as a future task on asana.